src/tests/ftest/io/nvme_io.py

```diff
@@ -71,6 +74,9 @@ def test_nvme_io(self):
                 self.pool.nvme_size.update(ior_param[1])
                 self.pool.create()
 
+                # Disable aggregation
+                self.pool.set_property()
```

Comment:
It would be good to make a note in the commit message and/or jira ticket as to why this is necessary and what it fixes.